### DISCUSSION TITLE
Roles and Permissions - Resizing Columns

### DISCUSSION THREAD

**Discussion by rfos** [directus/directus#12816] [Feature Requests]
When I'm adding new "Roles" I have no way to resize the columns "name", "users" and "description", then their names are not fully visible in the app.

Would be possible to add that functionality in the next release?

![2022-04-18 09_43_06-Settings](https://user-images.githubusercontent.com/217889/163774723-f27a450c-125c-44e9-a027-24b42e69711f.png)

Thank you

**Comment by rijkvanzanten** [comment]
Easy tweak, big quality of life improvement. Added in #12822
